crates/refunder/src/submitter.rs

```diff
@@ -55,7 +55,7 @@ impl Submitter {
             .eth()
             .transaction_count(self.account.address(), None)
             .await
-            .map_err(|err| anyhow!("Could not get latest nonce due to err: {:}", err))
+            .map_err(|err| anyhow!("Could not get latest nonce due to err: {err}"))
```

Comment:
I noticed you changed one log from `{err}` to `{err:?}` to get the error log with more details. Should this also happen here?

Reply:
The one I changed was logging an anyhow error, which should always be done with Debug printing to make it show the causes. This is a web3 error. I think it doesn't matter for these but haven't tested.